rwgame/RWGame.cpp

```diff
@@ -48,39 +49,60 @@ RWGame::RWGame(int argc, char* argv[])
 	bool fullscreen = false;
 	bool newgame = false;
 	bool test = false;
-    std::string startSave;
+	std::string startSave;
 	std::string benchFile;
 
-	for( int i = 1; i < argc; ++i )
+	// Define and parse command line options
+	namespace po = boost::program_options;
+	po::options_description desc("Available options");
+	desc.add_options()
+		("help",                                   "Show this help message")
+		("width,w",      po::value<size_t>(),      "Game resolution width in pixel")
+		("height,h",     po::value<size_t>(),      "Game resolution height in pixel")
+		("fullscreen,f",                           "Enable fullscreen mode")
+		("newgame",                                "Start a new game") // TODO
+		("test",                                   "Testmode") // TODO
```

Comment:
This interacts with `newgame`, it creates some vehicles & weapons instead of starting the game script.